Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
+source "https://rubygems.pkg.github.com/hubbado"
```

Comment:
ok, so the problem is here, if I understand it well. I don't have permission to this package.
So to make this gem public, you need remove this.